### PR TITLE
Partially rewrote the code of conduct page.

### PR DESCRIPTION
**Pull request by mxsasha**
This is a pretty big rewrite of the CoC page. However, the essence if our CoC is still the same.

Most significant changes:
* Removed the distinction between a long a short version, as they were virtually the same length.
* Added clarification on which venues the CoC applies to.
* Added reasons for having a CoC, with which I hope to both encourage reporting, and improve understanding.
* Expanded the list of specifically listed classes as used by the DSF.
* Added clarifications on what information to include in a report, and encouraging people not to feel like a burden and encouraging a report even if people are unsure.
* Added basic possible actions.
* Added note that we will not generally contact the police without the reporter's consent.
* Fixed a bunch of language inconsistencies.

### CODE REVIEW

**Comment by raphaelm** [comment]
I like it a lot.

File: conduct.md [hunk 1]
```diff
@@ -4,66 +4,85 @@ layout: page
...
-  religion.
-* We do not tolerate harassment of conference participants in any form.
-* Sexual language and imagery are not appropriate for any conference venue, including talks, evening
-  events, and the conference hotel.
-* Be kind to others. Do not insult or put down other attendees. Behave professionally. Don’t publish
-  individual photographs of people without their consent. Remember that harassment and sexist,
-  racist, or exclusionary jokes are not appropriate for DjangoCon Europe.
-* Conference participants violating these rules may be sanctioned or expelled from the conference
-  without a refund at the discretion of the conference organizers.
+* Helping everyone feel safe and included. Many attendees are first timers, or have had poor experiences in other events. We want to set the expectation that harassment and other unpleasant behaviour are not acceptable. So that if people do have an unpleasant experience, they know that’s not the norm.
```
Comment (rixx) [inline conduct.md]: "… they know it is neither the norm nor acceptable to us as a community", maybe?

File: conduct.md [hunk 2]
```diff
@@ -4,66 +4,85 @@ layout: page
 
 ## Code of Conduct
 
-DjangoCon Europe aims to be a welcoming event where people meet in a friendly environment.
-Accordingly, all participants are expected to show respect and courtesy to other participants
-throughout the conference.
+All delegates/attendees, speakers, exhibitors, organizers and
```
Comment (rixx) [inline conduct.md]: "Everybody who participates in DjangoCon Europe in one way or another, including …" might make this even more clear, and remove any roles we might forget now or that will be introduced later on.

File: conduct.md [hunk 3]
```diff
@@ -4,66 +4,85 @@ layout: page
...
+volunteers at DjangoCon Europe 2018 are required to conform to the following Code of Conduct. Organizers will enforce this code before and throughout the event (including related activities such as the social events).
```
Comment (rixx) [inline conduct.md]: "They will introduce a Code of Conduct team that will be primarily responsible and will follow the procedures listed below.", maybe, to make clear that this will not only be the organizers in general, but that there will be a dedicated point of contact.

File: conduct.md [hunk 4]
```diff
@@ -4,66 +4,85 @@ layout: page
...
 
-To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and
-volunteers at DjangoCon Europe 2018 event are required to conform to the following Code of Conduct.
-Organizers will enforce this code before and throughout the event (including related activites such
-as the social events).
+## Why do we have a Code of Conduct?
 
-### In short
+Our goals with having this CoC are:
```
Comment (rixx) [inline conduct.md]: I'd prefer calling it "Code of Conduct" throughout, as CoC is an abbreviation not commonly known to all.

File: conduct.md [hunk 5]
```diff
@@ -4,66 +4,85 @@ layout: page
...
-* We do not tolerate harassment of conference participants in any form.
-* Sexual language and imagery are not appropriate for any conference venue, including talks, evening
-  events, and the conference hotel.
-* Be kind to others. Do not insult or put down other attendees. Behave professionally. Don’t publish
-  individual photographs of people without their consent. Remember that harassment and sexist,
-  racist, or exclusionary jokes are not appropriate for DjangoCon Europe.
-* Conference participants violating these rules may be sanctioned or expelled from the conference
-  without a refund at the discretion of the conference organizers.
+* Helping everyone feel safe and included. Many attendees are first timers, or have had poor experiences in other events. We want to set the expectation that harassment and other unpleasant behaviour are not acceptable. So that if people do have an unpleasant experience, they know that’s not the norm.
+* Helping to build trust that if an incident is reported, we will not respond with victim blaming, and that we will do a serious investigation. Even, for example, if the incident concerns someone in a position of power.
```
Comment (rixx) [inline conduct.md]: ", and that we will start a thorough investigation." sounds a bit better to me.

File: conduct.md [hunk 6]
```diff
@@ -4,66 +4,85 @@ layout: page
...
+* Informing everyone of the expected behaviour. We are a diverse community, and so it’s valuable to have the CoC as a basis for the expectations from everyone in our community.
```
Comment (rixx) [inline conduct.md]: "We are a diverse community, and having a Code of Conduct makes the expectations of everybody's behaviour explicit and transparent." maybe?

File: conduct.md [hunk 7]
```diff
@@ -4,66 +4,85 @@ layout: page
...
+* Having a framework for report handling. The CoC is the first starting point for dealing with a report, assessing whether the CoC was violated, and what action should be taken.
```
Comment (rixx) [inline conduct.md]: "Providing a framework for handling reports. The Code of Conduct contains not only expected behaviour, but also how we deal with reports, assessing whether the Code of Conduct was violated, and what actions will be taken.", maybe?

File: conduct.md [hunk 8]
```diff
@@ -4,66 +4,85 @@ layout: page
...
+
+Please do not feel like you may be a burden to us by reporting incidents. Even if you happen to report multiple incidents during the conference. We rather consider reports an opportunity for us to act: by knowing about an incident, we can act on it, and often prevent it from continuing or repeating. But if we don’t know, we can’t take action.
+
+If you are not sure whether the situation was a Code of Conduct violation, or whether it applied to that particular space, we encourage you to still report it. We would much rather have a few extra reports where we decide to take no action, rather than miss a report of an actual violation. We do not look negatively on you if we find the incident is not a violation. And knowing about incidents that are not violations, or happen outside our spaces, can also help us to improve the Code of Conduct or the processes surrounding it.
 
-If you’re currently afraid of your physical safety or are in danger, contact local law enforcement
-in Germany:
+In your report please include, when possible:
+
+* Your contact info (so we can get in touch with you)
```
Comment (rixx) [inline conduct.md:66]: "If you feel unsafe reporting in person, you may choose one person to represent you. In this case, we'd need their contact information, but we'd ask you to make clear that this person represents you." ?

**Review by rixx — COMMENTED** [review]
Thank you for these changes!

File: conduct.md (hunk 7 above)
Comment (mxsasha) [reply conduct.md]: I reworded this slightly, but did not add "how we deal with reports" or "what actions will be taken". The former belongs in a separate guide, to be written, and for the latter I intentionally included a non-limitative list later on, as we may need other actions.

File: conduct.md (hunk 5 above)
Comment (mxsasha) [reply conduct.md]: 👍

File: conduct.md (hunk 1 above)
Comment (mxsasha) [reply conduct.md]: 👍

File: conduct.md (hunk 4 above)
Comment (mxsasha) [reply conduct.md]: I've introduced the abbreviation explicitly in the first paragraph. In other uses, I've gone for CoC only where it would otherwise be very repetitive.

File: conduct.md (hunk 3 above)
Comment (mxsasha) [reply conduct.md]: 👍

File: conduct.md (hunk 2 above)
Comment (mxsasha) [reply conduct.md]: 👍

File: conduct.md (hunk 6 above)
Comment (mxsasha) [reply conduct.md]: 👍

File: conduct.md (hunk 8 above)
Comment (mxsasha) [reply conduct.md:66]: 👍
